Soumikd mailscape v3

shalearkane opened the pull request:
Additions : 
starMail function created.
created a basic UI to attach multiple files and send emails.

Changes : 
renamed downloadMail to getMail.
adjusted sendMail function to be used by the UI

shalearkane committed:
Initial code to authorize and send an email from cmdline.
Extra features : get credentials from file to prevent leaking on git repo.
Changes from official instructions : use SSL instead of TLS.

shalearkane committed:
Changes : added a file picker which can choose multiple files.

shalearkane committed:
Additions : created a new class for retrieving mail.
Changes : moved getCredentials to a new class
Improvement : removed the unneeded String results[] and made the no. of emails to be retrieved be known from the function call.

shalearkane committed:
Additions : starMail function created
Changes : renamed downloadMail to getMail

shalearkane committed:
Additions :
starMail function created.
created a basic UI to attach multiple files and send emails.

Changes : renamed downloadMail to getMail.
adjusted sendMail function to be used by the UI

srijan2002 reviewed (commented):
PR Labelled as hacktoberfest-accepted

shalearkane committed:
Additions : Created Tabbed UI for sending and receiving mail.
Changes : get messages from Gmail by UID, therefore reducing time to fetch attachments.
